components/Sponsors.jsx

```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Heading from "./Heading";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center";
+const HEADING_CONTAINER = "flex flex-row justify-center mb-10";
+const SPONSORS_LEVEL_CONTAINER = "flex flex-row justify-center items-center my-5";
+const CONTENT_CONTAINER = "my-10 font-medium max-w-md text-center";

```

Comment:
This container looks wider in figma. Maybe `font-base w-2/3`?

Reply:
<img width="1276" alt="Screen Shot 2022-08-08 at 3 32 37 PM" src="https://user-images.githubusercontent.com/41935461/183525807-744c3268-7dd1-4b53-ac63-13f3bc008244.png">
That's too wide on my screen. I'll play around with it a bit